cookiecutter/main.py

```diff
@@ -70,8 +70,8 @@ def expand_abbreviations(template, config_dict):
 
 
 def cookiecutter(
-        template,
-        checkout=None, no_input=False, extra_context=None, replay=False):
+        template, checkout=None, no_input=False, extra_context=None,
+        replay=False, **kwargs):
```

Comment:
@pydanny @hackebrot It looks like misinterpreted the previous comments on using kwargs and thought that I should use the ``**kwargs`` structure. My bad. Will fix that :)